baybe/utils/optional.py

```diff
@@ -0,0 +1,56 @@
+"""Utilities for handling optional dependencies."""
+
+import importlib
+import importlib.util
+import warnings
+from typing import Any, Literal, Optional
+
+
+def import_optional_module(
+    name: str,
+    attribute: Optional[str] = None,
+    error: Literal["raise", "warn", "ignore"] = "raise",
+) -> Optional[Any]:
+    """Import an optional module.
+
+    Args:
+        name: The name of the module.
+        attribute: The name of an attribute to import from the module.
+        error: How to handle errors.
```

Comment:
We had some problems with rendering lists within Args. Please build the documentation locally (via "tox -e docs-py311") and just have a look at the created output and verify that it looks good. If not, please reformulate this, either via a note or by making this to a text block.

Reply:
This is what I see. 

![Screenshot 2024-01-22 at 10 48 48 AM](https://github.com/emdgroup/baybe/assets/5112312/b81143f0-5a7d-4c1c-b45b-d039a83b0e8a)

Reply:
I get your point. It's removed now.